### ISSUE TITLE
Add unit tests for Review and Draft 

### ISSUE THREAD

**Issue by CZagrobelny** [CZagrobelny/new_sanctuary_asylum#108]
For Review
- add a unit test to cover this method: authored_by?(user)

For Draft
- add a unit test to cover this method: status_string

**Comment by Pchao93** [comment]
I'd like to claim this issue, but when I created a feature branch to push up, I am getting a 'permission denied' error.

**Comment by CZagrobelny** [comment]
@Pchao93 my apologies, I thought I had updated the Contributing instructions to specify that everyone needs to fork the repo.  Updating it now.  If you haven't forked a repo to contribute before, here are some instructions:  https://guides.github.com/activities/forking/
In the past, when I have a branch ready without realizing I needed to fork the repo, I've used the following quick fix:
1.  Fork the repository on github (this is the first step in  https://guides.github.com/activities/forking/)
2.  Copy the 'Clone with SSH' link for the repo you just forked
3.  git remote add forked [Clone with SSH link for the forked repo]
This creates a remote for the forked repo that I can push to:
git push forked branch-name
Thanks!
